### PR TITLE
chore: improve documentation

### PR DESCRIPTION
**Pull request by roalonso**
- Do not display base AuctionRequest definition.
- Make `products` required for sponsored listings (and length at least one).
- Fix auction request example.
- Add missing descriptions.

### CODE REVIEW

File: topsort-api-v2.yml [hunk 1]
```diff
@@ -82,6 +79,22 @@ paths:
...
               required:
                 - auctions
+              example:
+                - type: listings
+                  slots: 2
+                  products:
+                    - productId: p_OAmRYLx9qN51
+                  # category:
+                  #   id: c_yogurt
+                  # searchQuery: Noosa Peach
```
Comment (PabloReszczynski) [inline topsort-api-v2.yml:89]: If `slots` is 2, then it should have at least 2 products in the product list

**Review by PabloReszczynski — CHANGES_REQUESTED** [review]
Just some changes around examples

File: topsort-api-v2.yml (hunk 1 above)
Comment (roalonso) [reply topsort-api-v2.yml:89]: Good point.

**Review by PabloReszczynski — APPROVED** [review]
👍

**Pull request merged**